checker/jdk/Makefile.jdk

```diff
@@ -62,7 +62,7 @@ validate: classes $(DIFFS)
 # does not exist.  So long as the variables are not used on such a make
 # invocation, this doesn't cause a problem, but it is irritating (and a bit
 # sloppy).
-BUILD_CLASSES = $(shell find build/java build/javax -name '*.class')
+BUILD_CLASSES = $(shell find build/java build/javax build/org/w3c build/org/xml -name '*.class')
```

Comment:
The `-regex` options does not work the same with BSD find.  I switched back to duplicating the folders, so that the annotated JDK can be built on Macs without installing GNU find.